### PR TITLE
update scripts

### PR DESCRIPTION
**Pull request by furkansenharputlu**

### CODE REVIEW

File: scripts/dev-test-rc.sh [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+HAZELCAST_TEST_VERSION="3.10"
 HAZELCAST_VERSION=${HZ_VERSION}
 HAZELCAST_ENTERPRISE_VERSION=${HZ_VERSION}
-HAZELCAST_RC_VERSION="0.3-SNAPSHOT"
+HAZELCAST_RC_VERSION="0.4-SNAPSHOT"
 SNAPSHOT_REPO="https://oss.sonatype.org/content/repositories/snapshots"
 RELEASE_REPO="http://repo1.maven.apache.org/maven2"
 ENTERPRISE_RELEASE_REPO="https://repository-hazelcast-l337.forge.cloudbees.com/release/"
-ENTERPRISE_SNAPSHOT_REPO="https://repository-hazelcast-l337.forge.cloudbees.com/snapshot/"
+ENTERPRISE_SNAPSHOT_REgiPO="https://repository-hazelcast-l337.forge.cloudbees.com/snapshot/"
```
Comment (mustafaiman) [inline scripts/dev-test-rc.sh]: typo -> `REgiPO`

**Comment by furkansenharputlu** [comment]
verify

**Comment by furkansenharputlu** [comment]
verify

**Comment by mustafaiman** [comment]
These tests failed:
```
1) NearCachedMap invalidate on change=true get returns null if the entry was removed by another client:
15:00:34 
15:00:34       AssertionError: expected 1 to equal 0
15:00:34       + expected - actual
15:00:34 
15:00:34       -1
15:00:34       +0
15:00:34       
15:00:34       at expectStats (test/map/NearCachedMapTest.js:81:43)
15:00:34       at test/map/NearCachedMapTest.js:135:21
15:00:34       at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
15:00:34       at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:512:31)
15:00:34       at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:569:18)
15:00:34       at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)
15:00:34       at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:693:18)
15:00:34       at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
15:00:34       at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
15:00:34       at Immediate.Async.drainQueues [as _onImmediate] (node_modules/bluebird/js/release/async.js:17:14)
15:00:34 
15:00:34   2) SSL Client Authentication Test via programmatic configuration "after each" hook for "ma:required, server does not know client, client knows server should fail":
15:00:34      Error: Timeout of 8000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```
I am running the test suite again to see if those two are just flaky tests or this PR causes the two tests fail consistently.

**Comment by mustafaiman** [comment]
verify

**Comment by mustafaiman** [comment]
I am thinking there is a recurring pattern. This test failed again
```
 1) SSL Client Authentication Test via programmatic configuration "after each" hook for "ma:required, server does not know client, client knows server should fail":
15:31:10      Error: Timeout of 8000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```

Let's find why this starts to fail when we change the server version before merging this PR

**Pull request merged**